### PR TITLE
	Fixed the symfony data collector when an exception is throwed 

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

File: src/Bridge/Symfony/Resources/views/data_collector/seo-override.html.twig [hunk 1]
```diff
@@ -20,7 +20,7 @@
             {{ macros.icon }}
         </span>
 
-        <span class="sf-toolbar-value sf-toolbar-status sf-toolbar-status-{{ collector.status == constant('Joli\\SeoOverride\\Bridge\\Symfony\\DataCollector\\SeoOverrideDataCollector::STATUS_MATCHED') ? 'green' : 'red' }}">{{ collector.statusLabel }}</span>
+        <span class="sf-toolbar-value sf-toolbar-status sf-toolbar-status-{{ collector.status == constant('STATUS_MATCHED', collector) ? 'green' : 'red' }}">{{ collector.statusLabel }}</span>
```
Comment (lyrixx) [inline src/Bridge/Symfony/Resources/views/data_collector/seo-override.html.twig]: just because I can ;)

Comment (damienalexandre) [inline src/Bridge/Symfony/Resources/views/data_collector/seo-override.html.twig]: O_O

File: src/Bridge/Symfony/DataCollector/SeoManager.php [hunk 2]
```diff
@@ -36,6 +36,7 @@ public function __construct(array $fetchers, array $domains, Seo $seo = null, ar
         $this->data['fetchersMapping'] = $fetchersMapping;
         $this->data['fetchers'] = [];
         $this->data['domains'] = array_keys($domains);
+        $this->data['status'] = SeoOverrideDataCollector::STATUS_DEFAULT;
```
Comment (pyrech) [inline src/Bridge/Symfony/DataCollector/SeoManager.php]: Could you remove the set done in updateSeo method? :kissing_heart:

Comment (lyrixx) [inline src/Bridge/Symfony/DataCollector/SeoManager.php]: no

Comment (lyrixx) [inline src/Bridge/Symfony/DataCollector/SeoManager.php]: done ;)

**Comment by pyrech** [comment]
Thanks :ok_hand:

**Pull request merged**